examples/vector_tile_raster_2d.html

```diff
@@ -39,8 +39,8 @@
                     // We want to keep the rotation disabled, to only have a view from the top
                     enableRotation: false,
                     // Faster zoom in/out speed
-                    zoomInFactor: 0.5,
-                    zoomOutFactor: 0.5,
+                    zoomInFactor: 3,
```

Comment:
may be remove this setting in examples and use default value?

Reply:
I removed them here but left them in the other example so that there is a track on how to use these parameters.